docs/modules/usage/llms/azure-llms.md

```diff
@@ -1,55 +1,42 @@
 # Azure OpenAI LLM
 
-## Completion
-
 OpenHands uses LiteLLM for completion calls. You can find their documentation on Azure [here](https://docs.litellm.ai/docs/providers/azure).
 
-### Azure openai configs
+## Azure OpenAI Configuration
 
-When running the OpenHands Docker image, you'll need to set the following environment variables using `-e`:
+When running OpenHands, you'll need to set the following environment variable using `-e` in the
+[docker run command](/modules/usage/getting-started#installation):
 
 ```
-LLM_BASE_URL="<azure-api-base-url>"          # e.g. "https://openai-gpt-4-test-v-1.openai.azure.com/"
-LLM_API_KEY="<azure-api-key>"
-LLM_MODEL="azure/<your-gpt-deployment-name>"
 LLM_API_VERSION="<api-version>"              # e.g. "2024-02-15-preview"
 ```
 
 Example:
 ```bash
-docker run -it \
---pull=always \
--e SANDBOX_USER_ID=$(id -u) \
--e WORKSPACE_MOUNT_PATH=$WORKSPACE_BASE \
--e LLM_BASE_URL="x.openai.azure.com" \
--e LLM_API_VERSION="2024-02-15-preview" \
--v $WORKSPACE_BASE:/opt/workspace_base \
--v /var/run/docker.sock:/var/run/docker.sock \
--p 3000:3000 \
---add-host host.docker.internal:host-gateway \
---name openhands-app-$(date +%Y%m%d%H%M%S) \
-ghcr.io/all-hands-ai/openhands:main
+docker run -it --pull=always \
+    -e LLM_API_VERSION="2024-02-15-preview"
+    ...
 ```
 
-You can also set the model and API key in the OpenHands UI through the Settings.
+Then set the following in the OpenHands UI through the Settings:
+* `LLM Provider` to `Azure`
+* `LLM Model` to the model you will be using.
+[Visit **here** to see a list of Azure models that LiteLLM supports.](https://docs.litellm.ai/docs/providers/azure#azure-openai-chat-completion-models)
+If the model is not in the list, toggle `Advanced Options`, and enter it in `Custom Model` (i.e. azure/&lt;model-name&gt;).
+* `API Key`
 
 :::note
-You can find your ChatGPT deployment name on the deployments page in Azure. It could be the same with the chat model
-name (e.g. 'GPT4-1106-preview'), by default or initially set, but it doesn't have to be the same. Run OpenHands,
-and when you load it in the browser, go to Settings and set model as above: "azure/&lt;your-actual-gpt-deployment-name&gt;".
-If it's not in the list, you can open the Settings modal, switch to "Custom Model", and enter your model name.
+You can find your ChatGPT deployment name on the deployments page in Azure.
```

Comment:
We say "deployment" here, it would be good to note at "model" above, that it is "deployment name"... From my recollection, it needs to be deployment name. Users can name their deployments of a model on Azure as they like, so they might name them with the standard model name of the model they deployed, or they might not. In the first case, it will match what we show, otherwise it won't, and they need to enter it manually in the box.

Reply:
Ah good to know. Will change.

Reply:
Made general changes to doc flow.